From 4ead8e2461ba0b86ff699149b624b35a46919f07 Mon Sep 17 00:00:00 2001
From: "Matthieu Baerts (NGI0)" <matttbe@kernel.org>
Date: Wed, 20 Nov 2024 15:20:28 +0100
Subject: [PATCH 4/4] scripts: use short stat options

Busybox's stat doesn't understand the long options:

  stat: unrecognized option: dereference

But it does support '-L'.

Signed-off-by: Matthieu Baerts (NGI0) <matttbe@kernel.org>
---
 scripts/check-permissions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/check-permissions b/scripts/check-permissions
index 879661b..a8dfedb 100755
--- a/scripts/check-permissions
+++ b/scripts/check-permissions
@@ -19,7 +19,7 @@ exit_status=0
 
 for p in $@; do
     # Access rights in human readable form (e.g. "drwxrwxr-x")
-    perms=`stat --dereference --format=%A $p`
+    perms=`stat -L -c %A $p`
 
     # The write mode for "others".
     other_write=`echo $perms | sed -e 's/.*\(.\).$/\1/'`
-- 
2.45.2

